silk/tests/testcase.py

```diff
@@ -46,15 +48,27 @@
 PING_RECEIVED = "pings_received"
 PING_ROUND_TRIP_TIME = "ping_rtt"
 
+DEFAULT_DISPATCHER_HOST = "localhost"
```

Comment:
```suggestion
DEFAULT_OTNS_HOST = "localhost"
```
dispatcher is a concept of OTNS, but not necessarily visible to silk users?